### DISCUSSION TITLE
How can I use a trained model for new predictions?

### DISCUSSION THREAD

**Discussion by anshumansinha16** [lululxvi/deepxde#916] [Q&A]
I tried the DeepOnet operator learning example according to the dataset generated from grf [link](https://deepxde.readthedocs.io/en/latest/demos/operator/antiderivative_aligned.html) . Now I want to apply the trained model on a known pair of function like cos(x) and sin(x) ; I'll generate my input data-field data as `x = np.arange(0,4*np.pi,0.1)` ; `u(y) =` `y = np.cos(x)` now I want to plot the anti-derivate of my u(y) and compare it with `y = np.cos(x)`. 

What additional code shall I implement in order to do so!

**Comment by lululxvi** [comment]
See FAQ Q: How can I save a trained model and then load the model later?
